sapp/ui/frontend/src/Runs.js

```diff
@@ -64,6 +64,10 @@ function Run(props: $ReadOnly<{run: RunDescription}>): React$Node {
     Modal.error({title: 'Unable to delete run ', content: deleteError.toString()});
   }
 
+  const clearFilters = () => {
```

Comment:
Can we move this function into `Filter.js`, since that's where other filter related code lives (e.g. [`loadFilter()`](https://github.com/facebook/sapp/blob/fd601478a34156eb1bf0aecd63debc98e140bcac/sapp/ui/frontend/src/Filter.js#L915))

Reply:
Sure!